functions.php

```diff
@@ -66,7 +66,7 @@ function wp_auth0_is_current_login_action( array $actions ) {
 /**
  * Generate a valid WordPress login override URL, if plugin settings allow.
  *
- * @param null $login_url - An existing URL to modify; default is wp-login.php.
+ * @param string|null $login_url - An existing URL to modify; default is wp-login.php.
  *
  * @return string
```

Comment:
Shouldn't this be a string or false? given you've enabled strict checking. I see you do this for params, so should be consistent.

Reply:
I know it seems strange, it's just an odd PHPism! Traditionally the community used `null` defaults or to pass undefined params for functions, as opposed expecting bools. I suspect this is because null has long had a performance edge over other falsy values in PHP when it comes to comparisons, but that isn't really the case anymore in modern versions

Now that typehinting has been added to the language and people are being a bit more thoughtful about their API desig there's a lot less of this happening, but unfortunately I can't add typehinting to this library quite yet as it would be a breaking change

In these particular cases, since we're not doing any comparisons and any falsy value works, I could just default these to empty strings to simply our docblocks for now